examples/demo.rs

```diff
@@ -157,7 +157,7 @@ async fn run(event_loop: EventLoop<()>, window: Window) {
                     });
                     // ... or a scope object
                     {
-                        wgpu_profiler::scope::Scope::start("fractal 1", &mut profiler, &mut rpass, &device);
+                        let mut rpass = wgpu_profiler::scope::Scope::start("fractal 1", &mut profiler, &mut rpass, &device);
```

Comment:
ouch. The fact that this works is a bit troublesome. I think what this means is that ::start should be `#[must_use]`